examples/add_colormaps.py

```diff
@@ -0,0 +1,31 @@
+"""
+Add named or unnamed vispy colormaps to existing layers.
+"""
+
+import numpy as np
+import vispy.color
+from skimage import data
+import napari.util
+
+histo = data.astronaut() / 255
+
+rch, gch, bch = np.transpose(histo, (2, 0, 1))
+
+red = vispy.color.Colormap([[0., 0., 0.], [1., 0., 0.]])
+green = vispy.color.Colormap([[0., 0., 0.], [0., 1., 0.]])
+blue = vispy.color.Colormap([[0., 0., 0.], [0., 0., 1.]])
+
+with napari.util.app_context():
+    v = napari.Viewer()
+
+    rlayer = v.add_image(rch)
+    rlayer.blending = 'additive'
+    rlayer.Colormap = 'red', red
```

Comment:
I think this needs to be `rlayer.colormap`, not `rlayer.Colormap`. If this is changed the string `red` appears in the dropdown menu and the image no longer looks washed out, but looks perfect rgb

Reply:
AAAAH.

Why review is good. =)